src/main/python/resources/annotaters/vep/motifName.py

```diff
@@ -0,0 +1,17 @@
+import pandas as pd
+from src.main.python.resources.annotaters.vep.template import Template
+import numpy as np
+
+
+class MotifName(Template):
+    def __init__(self):
+        super(MotifName, self).__init__(
+            name='MOTIF_NAME',
+            usable=True
+        )
+
+    def process(self, dataframe: pd.DataFrame):
+        dataframe['motifEName'] = dataframe[self.name]
+        dataframe['motifECount'] = np.where(dataframe[self.name].isna(), None,
+                                            '1')
+        return dataframe
```

Comment:
> Is there a functional reason for using `None` instead of `np.nan` (as it's one of the few examples in the test where `None` is expected instead of `np.nan`)?

This is due to Pandas not wanting to cooperate with setting NaN on a `object` (string) column, that's why it's `None` instead of `np.nan`. 

> Why use `None`/`1` instead of a boolean or `0`/`1`?

This is fixed in CAPICE imputing, because the annotator "shouldn't know what to do with an empty spot". This can be revised, as the impute value is `0`.

> Just out of curiosity: Why have a separate entry in the dataframe purely about whether `motifEName` is `na` or not (instead of simply checking whether `motifEName` is `na` where needed)?

It's a model feature whenever the variant has a `motifEName` or not. If the feature is beneficial is a different discussion, but it's a free feature, which couldn't hurt (or could it?)